pkg/gatherers/clusterconfig/jaeger_cr.go

```diff
@@ -38,6 +52,8 @@ func gatherJaegerCR(ctx context.Context, dynamicClient dynamic.Interface) ([]rec
 		return nil, []error{err}
 	}
 	var errs []error
+	// Limit the number of gathered jaegers.jaegertracing.io resources
+	var limit = 5
```

Comment:
This `limit` variable was declared at the package global scope and it could be changed anywhere, shadowing some variable in other scopes. Since it was just used here, I moved it to the right scope.